tasks/annotate-tag.js

```diff
@@ -0,0 +1,12 @@
+const shell = require('shelljs');
+const clipboardy = require('clipboardy');
+
+const [oldTag, newTag] = shell.exec('git tag', {silent: true}).tail({'-n': 2}).stdout.trim().split('\n');
+
+const compareText = `Changes: https://github.com/dideler/toggle-youtube-comments/compare/${oldTag}...${newTag}`
+
+// overwriting the lightweight tag with an annotated tag.
+shell.exec(`git tag --annotate --force ${newTag} ${newTag}^0 --message "${compareText}"`)
+
+// Copy markdown text.
+clipboardy.writeSync(`---\n\n${compareText}`);
```

Comment:
Does this inform the user that something has been copied to their clipboard?

Reply:
Ah, the current situation is not. It is better to display a message on the console. I'll try it.